integration-tests/src/test/java/uk/gov/di/authentication/api/UserInfoIntegrationTest.java

```diff
@@ -432,5 +464,15 @@ public UserInfoConfigurationService() {
         public boolean isIdentityEnabled() {
             return true;
         }
+
+        @Override
+        public boolean isTxmaAuditEnabled() {
+            return true;
```

Comment:
want this enabled by default given it's only going to a couple of environments?

Reply:
True - I didn't want to duplicate every test with new configuration, but maybe that's not avoidable.

Reply:
yes sorry that's test code